controller/controller.go

```diff
@@ -0,0 +1,46 @@
+package controller
+
+import (
+	"dashboard/model"
+	"database/sql"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+var db *sql.DB
+var err error
+
+//HandleSignUp func
+func HandleSignUp(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	var user model.User
+	stmt, err := db.Prepare("INSERT INTO users(Username,FirstName,LastName,Password,Gender,Country,Age,Email) VALUES(?)")

```

Comment:
Move this to separate package called db.

Reply:
should i move the prepare and exec statements also to db package?

Reply:
Yes, everything related to db operations (like insert, delete, select should be in db package) 
func insertUser(user *User) {
// prepare and exec.
}
and  you should call the function in handlers package like 
db.insertUser(&user)

Reply:
okay.